src/oop/user_defined_classes/medium/q5/QuestionText.md

```diff
@@ -0,0 +1,25 @@
+# oop-user_defined_classes-medium-q5
+
+## Java
+
+So many things in life cost money. Make a Java class called Bank that has one attribute 
+called fee and a default constructor that initializes the value to a constant. This class 
+also has another method called increase() that takes as input the number of months 
+from today so that it can calculate and return how much increase there is in the fee in the 
+future.
+
+
+
+
+## Bank
+
+The `Bank` class you are given has the following items:
+
+- Attributes:
+    - A `double` called **fee** 
+- Constructor:
```

Comment:
I think they can also code this constructor, it would be good for them to code one where it gets initialized to a value which is not passed as a parameter.